### PR TITLE
import into 7.5

### PR DESCRIPTION
**Pull request by D3Hunter**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-import-into.md [hunk 1]
```diff
@@ -33,6 +29,7 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)至少需要有 90 GiB 的可用空间。建议预留大于等于所需导入数据的存储空间，以保证最佳导入性能。
 - 一个导入任务只支持导入数据到一张目标表中。如需导入数据到多张目标表，需要在一张目标表导入完成后，再新建一个任务导入下一张目标表。
 - TiDB 集群升级期间不支持使用该语句。
+- 当使用全局排序时，编码后单个数据 KV 不能超过 32 MiB，单个索引 KV 不能超过 16MiB。
```
Comment (Benjamin2037) [inline sql-statements/sql-statement-import-into.md:32]:
```suggestion
- 当使用全局排序时，编码后单行数据总长度不能超过 32 MiB，单个索引 key 长度不能超过 16MiB。
```

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-import-into.md [hunk 2]
```diff
@@ -33,6 +29,7 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)至少需要有 90 GiB 的可用空间。建议预留大于等于所需导入数据的存储空间，以保证最佳导入性能。
 - 一个导入任务只支持导入数据到一张目标表中。如需导入数据到多张目标表，需要在一张目标表导入完成后，再新建一个任务导入下一张目标表。
 - TiDB 集群升级期间不支持使用该语句。
+- 当使用全局排序时，编码后单行数据总长度不能超过 32 MiB，单个索引 key 长度不能超过 16MiB。
```
[the remark this answers is not in the document]
Comment (D3Hunter) [reply sql-statements/sql-statement-import-into.md:33]: 任务会失败，msg 里会有

File: sql-statements/sql-statement-import-into.md [hunk 3]
```diff
@@ -155,6 +152,13 @@ SET 表达式左侧只能引用 `ColumnNameOrUserVarList` 中没有的列名。
 
 当开启[后端任务分布式框架](/tidb-distributed-execution-framework.md) 时，可通过 `IMPORT INTO` 的 `CLOUD_STORAGE_URI` 参数，或者使用系统变量 [`tidb_cloud_storage_uri`](/system-variables.md#tidb_cloud_storage_uri-从-v740-版本开始引入) 指定编码后的 KV 数据的目标存储地址来开启全局排序。注意目前仅支持使用 S3 作为全局排序存储地址。开启全局排序后，`IMPORT INTO` 会将编码后的 KV 数据写入云存储，并在云存储进行全局排序，之后再将全局排序后的索引数据和表数据并行导入到 TiKV，从而避免因 KV 重叠导致的问题，以提升导入的稳定性。
 
+全局排序对内存资源的使用较高，在数据导入开始前，建议先设置 [tidb_server_memory_limit_gc_trigger](/system-variables.md#tidb_server_memory_limit_gc_trigger-从-v640-版本开始引入) 和 [tidb_server_memory_limit](/system-variables.md#tidb_server_memory_limit-从-v640-版本开始引入) 两个变量，避免频繁触发 golang GC 从而影响导入效率:
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:155]:
```suggestion
全局排序对内存资源的使用较高，在数据导入开始前，建议先设置 [`tidb_server_memory_limit_gc_trigger`](/system-variables.md#tidb_server_memory_limit_gc_trigger-从-v640-版本开始引入) 和 [`tidb_server_memory_limit`](/system-variables.md#tidb_server_memory_limit-从-v640-版本开始引入) 两个变量，避免频繁触发 golang GC 从而影响导入效率:
```

File: sql-statements/sql-statement-import-into.md [hunk 4]
```diff
@@ -155,6 +152,13 @@ SET 表达式左侧只能引用 `ColumnNameOrUserVarList` 中没有的列名。
 
 当开启[后端任务分布式框架](/tidb-distributed-execution-framework.md) 时，可通过 `IMPORT INTO` 的 `CLOUD_STORAGE_URI` 参数，或者使用系统变量 [`tidb_cloud_storage_uri`](/system-variables.md#tidb_cloud_storage_uri-从-v740-版本开始引入) 指定编码后的 KV 数据的目标存储地址来开启全局排序。注意目前仅支持使用 S3 作为全局排序存储地址。开启全局排序后，`IMPORT INTO` 会将编码后的 KV 数据写入云存储，并在云存储进行全局排序，之后再将全局排序后的索引数据和表数据并行导入到 TiKV，从而避免因 KV 重叠导致的问题，以提升导入的稳定性。
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:153]:
```suggestion
当开启[后端任务分布式框架](/tidb-distributed-execution-framework.md) 时，可通过 `IMPORT INTO` 的 `CLOUD_STORAGE_URI` 参数，或者使用系统变量 [`tidb_cloud_storage_uri`](/system-variables.md#tidb_cloud_storage_uri-从-v740-版本开始引入) 指定编码后的 KV 数据的目标存储地址来开启[全局排序](/tidb-global-sort.md)。注意目前仅支持使用 S3 作为全局排序存储地址。开启全局排序后，`IMPORT INTO` 会将编码后的 KV 数据写入云存储，并在云存储进行全局排序，之后再将全局排序后的索引数据和表数据并行导入到 TiKV，从而避免因 KV 重叠导致的问题，以提升导入的稳定性。
```

File: tidb-global-sort.md [hunk 5]
```diff
@@ -41,6 +41,10 @@ TiDB 全局排序功能增强了数据导入和 DDL（数据定义语言）操
     SET GLOBAL tidb_cloud_storage_uri = 's3://my-bucket/test-data?role-arn=arn:aws:iam::888888888888:role/my-role'
     ```
 
+> **注意：**
+>
+> [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 也可使用 [`CLOUD_STORAGE_URI`](/sql-statements/sql-statement-import-into.md#withoptions) 参数来控制云存储路径。如果 [`tidb_cloud_storage_uri`](/system-variables.md#tidb_cloud_storage_uri-从-v740-版本开始引入) 和 `CLOUD_STORAGE_URI` 都设置了有效的目标存储地址，[`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 将以 `CLOUD_STORAGE_URI` 参数的配置为准。
```
Comment (qiancai) [inline tidb-global-sort.md:46]:
```suggestion
> [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 也可使用 [`CLOUD_STORAGE_URI`](/sql-statements/sql-statement-import-into.md#withoptions) 参数来控制云存储路径。如果 [`tidb_cloud_storage_uri`](/system-variables.md#tidb_cloud_storage_uri-从-v740-版本开始引入) 和 `CLOUD_STORAGE_URI` 都设置了有效的云存储路径，[`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 将以 `CLOUD_STORAGE_URI` 参数的配置为准。
```

**Review by hfxsd — APPROVED** [review]

File: sql-statements/sql-statement-import-into.md [hunk 6]
```diff
@@ -7,10 +7,6 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 
 `IMPORT INTO` 语句使用 TiDB Lightning 的[物理导入模式](/tidb-lightning/tidb-lightning-physical-import-mode.md)，用于将 `CSV`、`SQL`、`PARQUET` 等格式的数据导入到 TiDB 的一张空表中。
```
Comment (D3Hunter) [inline sql-statements/sql-statement-import-into.md:9]:
```suggestion
> **警告：**
>
> 目前该语句为实验特性，不建议在生产环境中使用。

```

File: sql-statements/sql-statement-import-into.md [hunk 7]
```diff
@@ -33,6 +33,7 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)至少需要有 90 GiB 的可用空间。建议预留大于等于所需导入数据的存储空间，以保证最佳导入性能。
 - 一个导入任务只支持导入数据到一张目标表中。如需导入数据到多张目标表，需要在一张目标表导入完成后，再新建一个任务导入下一张目标表。
 - TiDB 集群升级期间不支持使用该语句。
+- 当使用[全局排序](/tidb-global-sort.md)时，编码后单行数据的总长度不能超过 32 MiB，单个索引键的长度不能超过 16 MiB。
```
Comment (Frank945946) [inline sql-statements/sql-statement-import-into.md:37]:
```suggestion
当使用全局排序导入数据时，如果任务还未完成，把 TiDB 集群删除了，会在 S3 残留用于全局排序的临时数据，该场景需要手动介入删除这部分数据，以免增加 S3 的存储成本。
```

File: sql-statements/sql-statement-import-into.md [hunk 8]
```diff
@@ -33,6 +33,7 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)至少需要有 90 GiB 的可用空间。建议预留大于等于所需导入数据的存储空间，以保证最佳导入性能。
 - 一个导入任务只支持导入数据到一张目标表中。如需导入数据到多张目标表，需要在一张目标表导入完成后，再新建一个任务导入下一张目标表。
```
[the remark this answers is not in the document]
Comment (D3Hunter) [reply sql-statements/sql-statement-import-into.md:34]: 上面已经有了

File: sql-statements/sql-statement-import-into.md [hunk 9]
```diff
@@ -33,6 +33,7 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
...
 - TiDB 集群升级期间不支持使用该语句。
+- 当使用[全局排序](/tidb-global-sort.md)时，编码后单行数据的总长度不能超过 32 MiB，单个索引键的长度不能超过 16 MiB。
```
[the remark this answers is not in the document]
Comment (D3Hunter) [reply sql-statements/sql-statement-import-into.md:36]: 没法换算，跟主键有关

File: sql-statements/sql-statement-import-into.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (Frank945946) [reply sql-statements/sql-statement-import-into.md:30]: ok

**Review by Frank945946 — APPROVED** [review]

File: sql-statements/sql-statement-import-into.md [hunk 10]
```diff
@@ -33,6 +29,8 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)至少需要有 90 GiB 的可用空间。建议预留大于等于所需导入数据的存储空间，以保证最佳导入性能。
 - 一个导入任务只支持导入数据到一张目标表中。如需导入数据到多张目标表，需要在一张目标表导入完成后，再新建一个任务导入下一张目标表。
 - TiDB 集群升级期间不支持使用该语句。
+- 当使用[全局排序](/tidb-global-sort.md)时，单行数据的总长度不能超过 32 MiB。
+- 当使用全局排序导入数据时，如果任务还未完成，把 TiDB 集群删除了，会在 S3 残留用于全局排序的临时数据，该场景需要手动介入删除这部分数据，以免增加 S3 的存储成本。
```
Comment (Frank945946) [inline sql-statements/sql-statement-import-into.md:34]:
```suggestion
所需导入的数据不能存在主键、非空唯一索引冲突的记录，否则会导致任务失败。
```

**Review by Frank945946 — APPROVED** [review]

File: sql-statements/sql-statement-import-into.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (qiancai) [reply sql-statements/sql-statement-import-into.md:30]: L26 已经有了 （每个集群上同时只能有一个 `IMPORT INTO` 任务在运行。）

File: sql-statements/sql-statement-import-into.md [hunk 11]
```diff
@@ -33,6 +29,8 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
...
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:33]:
```suggestion
- 当使用全局排序导入数据时，如果 TiDB 集群在导入任务尚未完成时被删除了，Amazon S3 上可能会残留用于全局排序的临时数据。该场景需要手动删除这些数据，以免增加 S3 存储成本。
```

File: sql-statements/sql-statement-import-into.md [hunk 12]
```diff
@@ -33,7 +29,9 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)至少需要有 90 GiB 的可用空间。建议预留大于等于所需导入数据的存储空间，以保证最佳导入性能。
 - 一个导入任务只支持导入数据到一张目标表中。如需导入数据到多张目标表，需要在一张目标表导入完成后，再新建一个任务导入下一张目标表。
 - TiDB 集群升级期间不支持使用该语句。
-
+- 当使用[全局排序](/tidb-global-sort.md)时，单行数据的总长度不能超过 32 MiB。
+- 当使用全局排序导入数据时，如果 TiDB 集群在导入任务尚未完成时被删除了，Amazon S3 上可能会残留用于全局排序的临时数据。该场景需要手动删除这些数据，以免增加 S3 存储成本。
+- 所需导入的数据不能存在主键、非空唯一索引冲突的记录，否则会导致任务失败。
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:34]:
```suggestion
- 所需导入的数据不能存在主键、非空唯一索引冲突的记录，否则会导致任务失败。

```

File: sql-statements/sql-statement-import-into.md [hunk 13]
```diff
@@ -33,6 +29,9 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)至少需要有 90 GiB 的可用空间。建议预留大于等于所需导入数据的存储空间，以保证最佳导入性能。
 - 一个导入任务只支持导入数据到一张目标表中。如需导入数据到多张目标表，需要在一张目标表导入完成后，再新建一个任务导入下一张目标表。
 - TiDB 集群升级期间不支持使用该语句。
+- 当使用[全局排序](/tidb-global-sort.md)时，单行数据的总长度不能超过 32 MiB。
+- 当使用全局排序导入数据时，如果 TiDB 集群在导入任务尚未完成时被删除了，Amazon S3 上可能会残留用于全局排序的临时数据。该场景需要手动删除这些数据，以免增加 S3 存储成本。
+- 所需导入的数据不能存在主键、非空唯一索引冲突的记录，否则会导致任务失败。
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:34]:
```suggestion
- 所需导入的数据不能存在主键或非空唯一索引冲突的记录，否则会导致任务失败。
```

File: sql-statements/sql-statement-import-into.md [hunk 14]
```diff
@@ -33,6 +29,9 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)至少需要有 90 GiB 的可用空间。建议预留大于等于所需导入数据的存储空间，以保证最佳导入性能。
 - 一个导入任务只支持导入数据到一张目标表中。如需导入数据到多张目标表，需要在一张目标表导入完成后，再新建一个任务导入下一张目标表。
 - TiDB 集群升级期间不支持使用该语句。
+- 当使用[全局排序](/tidb-global-sort.md)时，单行数据的总长度不能超过 32 MiB。
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:32]:
```suggestion
- 当使用[全局排序](/tidb-global-sort.md)导入数据时，单行数据的总长度不能超过 32 MiB。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**